site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+# Concepts
+
+This section describes some essential concepts with which you will
+need to be familiar before moving on to the example deployment
+scenarios.
+
+> Author's Note: I have deliberately excluded any mention of
+> weave command syntax in this section
+
+## Peer
+
+A peer is a host on which you have installed weave.
+
+## Peer Name
+
+Peers in the weave network are identified by a 48-bit value formatted
+like an ethernet MAC address e.g. `01:23:45:67:89:ab`. This 'peer
+name' is used for various purposes:
+
+* Routing of packets between containers on the overlay network
+* Recording the origin peer of DNS entries
+* Recording ownership of IP address ranges
+
+Whilst the first two uses tolerate the peer name changing on router
+restarts, the third use means that address space can be 'lost' (e.g.
+recorded as being owned by a now non-existent peer name) and so it is
+desirable for the peer name to be as stable as possible across
+restarts. Consequently when the router is launched on a host it
+derives its peer name in order of preference:
+
+* From the command line; specifier is responsible for uniqueness and
+  stability
+* From the BIOS product UUID, which is generally stable across restarts
+  and unique across different physical hardware and cloned VMs
+* From a random value, unique across different physical hardware and
+  cloned VMs but not stable across restarts
+
+More important still than the stability constraint is uniqueness;
+address space lost due to a peer name change can be recovered with an
+administrative action, but if two or more peers share the same name
+chaos will ensue, including but not limited to double allocation of
+addresses and inability to route packets on the overlay network.
+
+The best strategy for assigning peer names depends on the type and
+method of your particular deployment and is discussed in more detail
+below.
+
+## Peer Discovery
+
+Peer discovery is a mechanism which allows peers to learn about new
+weave hosts from existing peers without being explicitly told. Peer
+discovery is enabled by default.
+
+## Network Partition
+
+A network partition is a transient condition whereby some arbitrary
+subsets of peers are unable to communicate with each other. Weave is
+designed to allow peers and their containers to make maximum safe
+progress under conditions of partition, healing automatically once the
+partition is over.
+
+## IP Address Manager (IPAM)
+
+IPAM is the subsystem responsible for dividing up a large contiguous
+block of IP addresses (known as the IP allocation range) amongst peers
+so that individual addresses may be uniquely assigned to containers
+anywhere on the overlay network.
+
+When a new network is formed an initial division of the IP allocation
+range must be made. Two (mutually exclusive) mechanisms with different
+tradeoffs are provided to perform this task: seeding and consensus.
+
+## Seeding
```

Comment:
Do you mean to make seeding and consensus sub-concepts of IPAM? What about electors & observers?